src/rollup-plugin.ts

```diff
@@ -1,7 +1,7 @@
 import * as fs from "fs";
 import * as fsp from "fs/promises";
 import * as path from "path";
-import * as fg from "fast-glob";
+import tg from "tiny-glob/sync"
```

Comment:
My bad on the typo, but I was actually suggesting https://www.npmjs.com/package/tinyglobby

SvelteKit does use tiny-glob, but we'll probably switch to tinyglobby because Vite, vitest, etc. are adopting tinyglobby

Reply:
Understood - will switch